snippets/comparison/migrating pages.md

```diff
@@ -55,10 +55,10 @@ export default function HomePage({ recentPosts }) {
 import HomePage from './HomePage';
 
 async function getPosts() {
-  const res = await fetch('https://...');
-  const posts = await res.json();
+  const posts = await fetch('https://...');
+  const data = await posts.json();
```

Comment:
This is not an improvement. The return of fetch is a response.

Reply:
This was changed to match how its done in the rest of the files and in the beta docs

Reply:
I can see `res` is still being used in the beta docs: https://beta.nextjs.org/docs/data-fetching/fetching#asyncawait-in-server-components